src/mlpack/methods/lars/lars.cpp

```diff
@@ -538,3 +538,23 @@ void LARS::CholeskyDelete(const size_t colToKill)
     matUtriCholFactor.shed_row(n);
   }
 }
+
+double LARS::ComputeError(const arma::mat& matX,
+                          const arma::rowvec& y,
+                          const bool rowMajor)
+{
+  double cost = 0.0;
+  arma::rowvec u;
+
+  if (rowMajor)
+    u = trans(matX * betaPath.back());
+
+  else
+    u = betaPath.back().t() * matX;
```

Comment:
If I am not wrong, we don't need to transpose `betapath.back() = beta` 
Basically,
`betapath.back()` has dimension (1 * m)  as it's a `rowvec`.
next if `X is column major` then it's dimension would be (m * n)
Now we need to have output (u) as (1 * n) as y would be of same dimension.
So I think the correct way is
u = betaPath.back() * matX as it would lead to (1 * n)

Let me know if I missed something

Reply:
Hey @walragatver thanks for suggestion. I think beta is a arma::vec which is colvec not rowvec as you can see same implemntation in the predict function.
https://github.com/mlpack/mlpack/blob/dcfcff4a91bf93f1486a5716b222263153fea1b4/src/mlpack/methods/lars/lars.cpp#L375
 I will add changes to train function thanks for suggestion

Reply:
Oh yeah, looks like I made minor mistake while reading.